karma/setup/page.ts

```diff
@@ -41,3 +31,21 @@ export function cleanupPage(): void {
 
     fixture.cleanup();
 }
+
+export async function setupPageNoRunClarity(done: DoneFn): Promise<void> {
```

Comment:
>setupPageNoRunClarity [](start = 22, length = 21)

What does this method name mean? I had a bit of trouble parsing the name.

Reply:
hmm I'm trying to express that it sets up the page without running Clarity...let me try to rephrase and see it if makes it easier to follow

---
In reply to: [310726526](https://github.com/microsoft/clarity-js/pull/146#discussion_r310726526) [](ancestors = 310726526)